src/main/kotlin/Calculator.kt

```diff
@@ -0,0 +1,52 @@
+class Calculator {
+    fun main(string: String?): Double {
+        if (string.isNullOrBlank()) {
+            throw IllegalArgumentException("입력값이 없습니다.")
+        } else {
```

Comment:
early return 이라는 패턴이 있습니니다! ( https://libertegrace.tistory.com/entry/Coding-Style-Return-Early-Pattern 이거 한번 읽어보셔도 좋을거 같아요! )
```suggestion
        }
```
throw 이후, 아래 라인들은 실행이 안될테니, else 문을 사용하지 않으셔도 좋을거같아요!